languages/ocaml.scm

```diff
@@ -189,6 +188,20 @@
   ]* @do_nothing
 )
 
+(
```

Comment:
It would be useful with some code examples here that demonstrate why the queries need to be like this.

Reply:
Also, would it make sense to merge these two patterns into one?

Reply:
I added a quite long comment with a code example. Was it the kind of explanation you expected?

Reply:
Well, I spoke too fast. The code written in the first bullet point failed the test, but only because the query 
```
(match_case
  "->"  @append_spaced_softline
  .
  (comment)* @do_nothing
)
```
did not exist. If I replace the 2 queries about arrows by those two, the current tests are passing.

Reply:
Great, thanks!